src/components/ShellView/ResultTable.vue

```diff
@@ -40,20 +40,39 @@
             </th>
           </tr>
         </thead>
-        <tbody>
-          <tr v-for="(row, i) in rows" :key="i">
-            <td v-for="(cell, j) in row" :key="j">
-              <ul v-if="Array.isArray(cell)" class="list-group">
-                <li v-for="(item, k) in cell" :key="k" class="list-group-item">
-                  <b>{{ item.name }}:</b> {{ item.value }}
+      <tbody>
+      <tr v-for="(row, i) in rows" :key="i">
+        <td v-for="(cell, j) in row" :key="j">
+          <div v-if="Array.isArray(cell)" class="list-container">
+            <ul class="list-group">
+              <li v-for="(item, k) in cell" :key="k" class="list-group-item">
+                <b>{{ item.name }}:</b> {{ item.value }}
+              </li>
+            </ul>
+          </div>
+          <div v-else-if="cell.hasOwnProperty('_label') && cell._label === 'RECURSIVE_REL'">
```

Comment:
This information could be implied from `j`. There is no need to store it repetitively.